libcodechecker/output_formatters.py

```diff
@@ -0,0 +1,186 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+Contains functions to format and pretty-print data from two-dimensional arrays.
+"""
+
+import json
+
+# The supported formats the users should specify. (This is not an exhaustive
+# list of ALL formats available.)
+USER_FORMATS = ['rows', 'table', 'csv', 'json']
```

Comment:
Could we use this list in the argument handler to show the user the selectable options
[here](https://github.com/Ericsson/codechecker/blob/master/libcodechecker/cmd/cmd_line_client.py#L497)?

How about checking the valid formats based on this list in the `twodim_to_str` function?
I did not found any usage of this list in the code.